src/components/pages/index.js

```diff
@@ -1,16 +1,26 @@
+import { useContext } from 'react';
 import MemberList from '@components/members-list';
 import classNames from '@components/pages/home-page.module.scss';
 import NewMemberList from '@components/new-member-list';
+import UserProfile from '@components/user-profile';
+import MemberRoleUpdate from '@components/member-role-update';
+import UserContext from '@store/user/user-context';
 
 const HomePage = () => {
+  const { showMemberRoleUpdateModal } = useContext(UserContext);
```

Comment:
I had a small confusion here, Is it better to define a function which returns `useContext(UserContext)` or call it in where ever we use it.

Reply:
ya that's a nice question